HW02_B_Mimic_starter.Rmd

```diff
@@ -91,7 +110,15 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
-
+mpg$class2 <- factor(mpg$class)
+ggplot(mpg, aes(cty, class)) +
+  geom_point(aes(color = class), position  = position_jitter(width = 0, height = 0.3)) +
+  scale_color_manual("class", values = brewer.pal(7, "Set2")) +
+  geom_boxplot(alpha = 0) +

```

Comment:
Good that you followed the order of the geom layers (boxplot on top of scatter plot)!